app/controllers/episodes_controller.rb

```diff
@@ -0,0 +1,28 @@
+class EpisodesController < ApplicationController
+  before_action :set_episode, only: [:show, :transcript]
+
+  def show
+    @html_id  = 'page'
+    @body_id  = 'podcast'
+    @editable = @episode
+    @title    = title_for :podcasts, @episode.name, :transcript
+  end
+
+  def transcript
+    @html_id  = 'page'
+    @body_id  = 'podcast'
+    @editable = @episode
+    @title    = title_for :podcasts, @episode.name, :transcript
+
+    render 'podcast/show'
+  end
+
+  private
+
+  def set_episode
+    @episode = Episode.where(id: params[:id])
```

Comment:
It seem like to me that you'll also need to include the `podcast_slug` in the lookup here, since the URL will be `podcast/:slug/episodes/:id`

And there will be multiple episode `1`. One for each podcast.

Reply:
Should I had something like `podcast_episode_id` ? Because right now, there is only the slug to work with and the id is the episode model id?

Reply:
That sounds like a good idea. 👍🏻

Reply:
I ended up adding `episode_number` to `Episode`, because it's not quite actually an `id`. But same general idea.